raddb/sites-available/dhcp

```diff
@@ -218,15 +208,11 @@ recv Request {
 
 	#  Or, allocate IPs from the DHCP pool in SQL. You may need to
 	#  set the pool name here if you haven't set it elsewhere.
-#	update control {
-#		&IP-Pool.Name := "local"
-#	}
+#	&control.IP-Pool.Name := "local"
 #	dhcp_sqlippool
 
 	if (ok) {
-		update reply {
-			&Your-IP-Address := "%{%{request.Requested-IP-Address}:-%{request.Client-IP-Address}}"
-		}
+		&reply.Your-IP-Address := "%{%{request.Requested-IP-Address}:-%{request.Client-IP-Address}}"
```

Comment:
I'd remove the `""` on the right side.  It's no longer necessary.  Just

`&reply.Your-IP-Address := %{%{request.Requested-IP-Address}:-%{request.Client-IP-Address}}`